pkg/cloud/nodelocal/nodelocal_storage.go

```diff
@@ -23,6 +23,8 @@ import (
 	"github.com/cockroachdb/cockroach/pkg/base"
```

Comment:
should external connection implementations live under the appropriate `pkg/cloud/*/` directories instead of `pkg/cloud/externalconn/*`?

Reply:
Maybe I'm misunderstanding but thats what it is today. Every provider specific package in `pkg/cloud` will define what their ConnectionDetails look like.

Reply:
Yup, not sure what I was reading!